ui/webui/src/components/localization/InstallationLanguage.jsx

```diff
@@ -178,9 +179,10 @@ class LanguageSelector extends React.Component {
                       key="group-common-languages"
                     >
                         {
-                            this.state.commonLocales.map(locale => {
-                                return createMenuItem(findLocaleWithId(locale), "option-common-");
-                            })
+                            this.state.commonLocales
+                                    .map(findLocaleWithId)
+                                    .filter(locale => locale)
+                                    .map(locale => createMenuItem(locale, "option-common-"))
```

Comment:
honestly feels that we iterate this a bit too much, but this page needs a bit of a re-design. Like it feels more sensible to do this once during initialisation.